themes/default/index.html

```diff
@@ -248,8 +248,6 @@
                   <td class="content-cell">
                     <% if (locals.title) { %>
```

Comment:
If you could please remove the `<% if (locals.title) { %>` since it will always be present from now on, that'd be great. =)